app/helpers/companies_helper.rb

```diff
@@ -1,8 +1,8 @@
 module CompaniesHelper
 
   def company_complete? company
-    company&.name.blank? ||
-    (company&.region.blank? && company&.old_region.blank?) ? false : true
+    return false if ! company
```

Comment:
I'm not sure this is used anywhere.  When I did a search in the code for `company_complete?`  I only found it being used in the specs and features.

We now only return `complete` companies from the database so there's no need to filter them in the views (via a helper).  So perhaps we can remove this method altogether?

If we do keep it, what about changing this to:

```
company.nil? ? false : company.complete?
```